Source/JavaScriptCore/runtime/TemporalObject.cpp

```diff
@@ -300,15 +300,15 @@ std::optional<TemporalUnit> temporalSmallestUnit(JSGlobalObject* globalObject, J
     return unitType;
 }
 
-static constexpr std::initializer_list<TemporalUnit> disallowedUnits[] = {
-    { },
-    { TemporalUnit::Hour, TemporalUnit::Minute, TemporalUnit::Second, TemporalUnit::Millisecond, TemporalUnit::Microsecond, TemporalUnit::Nanosecond },
-    { TemporalUnit::Year, TemporalUnit::Month, TemporalUnit::Week, TemporalUnit::Day }
-};
-
 // https://tc39.es/proposal-temporal/#sec-temporal-getdifferencesettings
 std::tuple<TemporalUnit, TemporalUnit, RoundingMode, double> extractDifferenceOptions(JSGlobalObject* globalObject, JSValue optionsValue, UnitGroup unitGroup, TemporalUnit defaultSmallestUnit, TemporalUnit defaultLargestUnit)
 {
+    static std::initializer_list<TemporalUnit> disallowedUnits[] = {
```

Comment:
I understand we can’t use `constexpr` because of a bug in the Windows compiler. Can we at least use `const`? That will lead to better performance on many platforms, not checking if the array needs to be initialized each time the function is called.